check/src/main/scala/scala/tools/jardiff/AsmTextifyRenderer.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbenc.com>
+ */
+// NOTE: This file has been partially copy/pasted from scala/jardiff.
```

Comment:
@retronym Have you by any chance published Jardiff to Maven Central? I couldn't find the binaries, so I had to copy/paste the source code.